internal/handler/handler.go

```diff
@@ -0,0 +1,91 @@
+package handler
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/ffddorf/unms-exporter/exporter"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/sirupsen/logrus"
+)
+
+type Handler struct {
```

Comment:
i think this should be a private type since `New` returns `http.Handler` and it would be an anti-pattern to create this without `New`.

Reply:
I think you're right.

Reply:
Done.